docs/faq/translate_ref.md

```diff
@@ -30,7 +30,7 @@ Augur will only translate genes which have 'CDS' as the feature name, and have a
                      /gene="ENV"
 ```
 
-Compare the [original Zika reference on Genbank](https://www.ncbi.nlm.nih.gov/nuccore/KX369547) to the [one used on Nextstrain](https://github.com/nextstrain/zika/blob/master/config/zika_reference.gb). Notice that the genes are designated by `CDS` instead of `mat_peptide` and have an entry for `/gene=` as well as `/product=`.
+Compare the [original Zika reference on Genbank](https://www.ncbi.nlm.nih.gov/nuccore/KX369547) to the [one used on Nextstrain](https://github.com/nextstrain/zika/blob/main/phylogenetic/defaults/zika_reference.gb). Notice that the genes are designated by `CDS` instead of `mat_peptide` and have an entry for `/gene=` as well as `/product=`.
```

Comment:
_(non-blocking)_

```suggestion
Compare the [original Zika reference on Genbank](https://www.ncbi.nlm.nih.gov/nuccore/KX369547) to the [one used on Nextstrain](https://github.com/nextstrain/zika/blob/-/phylogenetic/defaults/zika_reference.gb). Notice that the genes are designated by `CDS` instead of `mat_peptide` and have an entry for `/gene=` as well as `/product=`.
```

Use `-`/`HEAD`/`@` to future-proof against default branch name changes (previous discussion/explanation: https://github.com/nextstrain/auspice/pull/1660#discussion_r1155025654)